src/libpriv/rpmostree-core.h

```diff
@@ -35,6 +35,7 @@
 /* See http://lists.rpm.org/pipermail/rpm-maint/2017-October/006681.html */
 #define RPMOSTREE_RPMDB_LOCATION "usr/share/rpm"
 #define RPMOSTREE_SYSIMAGE_DIR "usr/lib/sysimage"
+#define RPMOSTREE_NEW_RPMDB_LOCATION RPMOSTREE_SYSIMAGE_DIR "/rpm"
```

Comment:
The funny thing with `NEW` is that it quickly becomes outdated. :) How about just e.g. `RPMOSTREE_SYSIMAGE_RPMDB_LOCATION`?